src/main.cpp

```diff
@@ -57,47 +57,47 @@ static struct options opt = {
     .max_log_files = 0,
 };
 
-static const struct option long_options[] = {
-    { "endpoints",              required_argument,  NULL,   'e' },
-    { "conf-file",              required_argument,  NULL,   'c' },
-    { "conf-dir" ,              required_argument,  NULL,   'd' },
-    { "report_msg_statistics",  no_argument,        NULL,   'r' },
-    { "tcp-port",               required_argument,  NULL,   't' },
-    { "tcp-endpoint",           required_argument,  NULL,   'p' },
-    { "log",                    required_argument,  NULL,   'l' },
-    { "debug-log-level",        required_argument,  NULL,   'g' },
-    { "verbose",                no_argument,        NULL,   'v' },
-    { "version",                no_argument,        NULL,   'V' },
-    { }
```

Comment:
this one probably deserves a comment on top to stop clang-format to make it horrible.